ops/testing.py

```diff
@@ -2931,23 +2931,49 @@ def add_layer(
         else:
             self._layers[label] = layer_obj
 
-    def _render_services(self) -> Dict[str, pebble.Service]:
-        services: Dict[str, pebble.Service] = {}
+    def _render_services(self) -> Dict[str, pebble.ServiceDict]:
+        services: Dict[str, pebble.ServiceDict] = {}
         for key in sorted(self._layers.keys()):
             layer = self._layers[key]
             for name, service in layer.services.items():
                 # TODO: (jam) 2021-04-07 have a way to merge existing services
-                services[name] = service
+                services[name] = service.to_dict()
         return services
 
+    def _render_checks(self) -> Dict[str, pebble.CheckDict]:
+        services: Dict[str, pebble.CheckDict] = {}
```

Comment:
This should be named `checks`.

Reply:
any reason why we don't have `services` as well then? I thought I'd make them private since that wasn't exposed to begin with

Reply:
Changing this and `log_targets` makes `test.test_testing.TestTestingPebbleClient.test_methods_match_pebble_client` fail because the public signature differs from pebble.Client. Do you want me to add these public methods to pebble.Client as well?

Reply:
Sorry if my original comment wasn't clear -- I didn't mean the `_render_checks` method name should change, but that the `services` variable name should be `checks`, because in this method we're building a dict of checks. Similar below for the variable name, `services` -> `log_targets`.